lib/shelley/src/Cardano/Wallet/Shelley/Network.hs

```diff
@@ -724,6 +727,9 @@ data NetworkLayerLog
     | MsgDestroyCursor ThreadId
     | MsgWillQueryRewardsForStake W.Coin
     | MsgFetchedNodePoolLsqData NodePoolLsqData
+    | MsgFetchedNodePoolLsqDataSummary Int Int
```

Comment:
Wish we had some kind of anonymous records / named-tuples (e.g. `(stakeDistrSize :: Int, rewardMapSize :: Int)`) for situations like these :/

Think a separate record-type is overkill and not sure `Quantity "pools in stakeDistr"` helps _that_ much. Or maybe the latter?

Reply:
> anonymous records / named-tuples

Yeah! One of the thing I really like in Elm <3